libcst/codemod/visitors/_apply_type_annotations.py

```diff
@@ -133,6 +134,11 @@ def _handle_Index(self, slice: cst.Index, node: cst.Subscript) -> cst.Subscript:
 
     def _handle_Subscript(self, node: cst.Subscript) -> cst.Subscript:
         slice = node.slice
+        if (
+            m.matches(node.value, m.Name())
+            and cst.ensure_type(node.value, cst.Name).value == "Type"
+        ):
```

Comment:
Use matcher to match the value to be simpler and easier to read.
```suggestion
        if m.matches(node.value, m.Name(value="type")):
```

Reply:
I did not know you could use this syntax. 😍